sklearn/cluster/_agglomerative.py

```diff
@@ -357,24 +357,24 @@ def linkage_tree(X, connectivity=None, n_clusters=None, linkage='complete',
 
     Parameters
     ----------
-    X : array, shape (n_samples, n_features)
+    X : ndarray of shape (n_samples, n_features)
```

Comment:
`X` can be a list. So, this should be `array-like` instead of `ndarray`.

Reply:
ok